src/get.rs

```diff
@@ -44,15 +44,15 @@ type ReplyInner = Option<Reply>;
 /// To check if `val` is still valid, you may use `z_X_check(&val)` (or `z_check(val)` if your compiler supports `_Generic`), which will return `true` if `val` is valid.
 #[cfg(target_arch = "x86_64")]
 #[repr(C, align(8))]
-pub struct z_owned_reply_t([u64; 23]);
+pub struct z_owned_reply_t([u64; 28]);
 
 #[cfg(target_arch = "aarch64")]
 #[repr(C, align(16))]
 pub struct z_owned_reply_t([u64; 24]);
 
 #[cfg(target_arch = "arm")]
 #[repr(C, align(8))]
-pub struct z_owned_reply_t([u64; 17]);
+pub struct z_owned_reply_t([u64; 17]); //TODO(sashacmc): adjust this size
```

Comment:
@milyin do you know how?

Reply:
```
#[cfg(target_arch = "x86_64")]
#[repr(C, align(8))]
pub struct z_owned_reply_t([u64; 28]);

#[cfg(target_arch = "aarch64")]
#[repr(C, align(16))]
pub struct z_owned_reply_t([u64; 30]);

#[cfg(target_arch = "arm")]
#[repr(C, align(8))]
pub struct z_owned_reply_t([u64; 19]);
```
The way to check for all architectures is to build target "crosscheck" with cmake. All prerequisites are here: [https://github.com/eclipse-zenoh/zenoh-c/blob/master/cmake/cross_build_check.cma](https://github.com/eclipse-zenoh/zenoh-c/blob/master/cmake/cross_build_check.cmake)